modules/claims_api/app/controllers/claims_api/v2/application_controller.rb

```diff
@@ -150,49 +87,6 @@ def get_benefits_documents_auth_token
         @auth_token ||= ClaimsApi::V2::BenefitsDocuments::Service.new.get_auth_token
       end
 
-      def build_target_veteran(veteran_id:, loa:) # rubocop:disable Metrics/MethodLength
-        target_veteran ||= ClaimsApi::Veteran.new(
-          mhv_icn: veteran_id,
-          loa:
-        )
-        # populate missing veteran attributes with their mpi record
-        found_record = target_veteran.mpi_record?(user_key: veteran_id)
-
-        unless found_record
-          claims_v2_logging('veteran_not_found', icn: veteran_id,
-                                                 message: 'Claims v2 Veteran record not found')
-          raise ::Common::Exceptions::ResourceNotFound.new(detail:
-            "Unable to locate Veteran's ID/ICN in Master Person Index (MPI). " \
-            'Please submit an issue at ask.va.gov or call 1-800-MyVA411 (800-698-2411) for assistance.')
-        end
-
-        mpi_profile = target_veteran&.mpi&.mvi_response&.profile || {}
-
-        if mpi_profile[:participant_id].blank?
-          claims_v2_logging('pid_not_found', icn: veteran_id,
-                                             message: 'Claims v2 Veteran PID not found')
-          raise ::Common::Exceptions::UnprocessableEntity.new(detail:
-            "Unable to locate Veteran's Participant ID in Master Person Index (MPI). " \
-            'Please submit an issue at ask.va.gov or call 1-800-MyVA411 (800-698-2411) for assistance.')
-        end
```

Comment:
this is stripped out entirely now with the move to `target_veteran.rb`. Any concerns with not having this check? participant ID is required for BGS, so 🤷

Reply:
I stand corrected. it's nested within the `mpi_profile_from`.